Lombiq.HelpfulExtensions.Tests.UI/Extensions/UITestContextExtensions.cs

```diff
@@ -0,0 +1,11 @@
+using Lombiq.Tests.UI.Extensions;
+using Lombiq.Tests.UI.Services;
+using System.Threading.Tasks;
+
+namespace Lombiq.HelpfulExtensions.Tests.UI.Extensions;
+
+public static class UITestContextExtensions
+{
+    public static Task GoToCreatePageAsync(this UITestContext context) =>
+        context.GoToRelativeUrlAsync("/Admin/Contents/ContentTypes/Page/Create", onlyIfNotAlreadyThere: false);
```

Comment:
There is already a `Lombiq.Tests.UI.Extensions.OrchardCoreDashboardUITestContextExtensions.CreateNewContentItemAsync` extension method.